12/alpine3.11/Dockerfile

```diff
@@ -33,25 +33,13 @@ RUN addgroup -g 1000 node \
         libgcc \
         linux-headers \
         make \
+        git \
+        bash \
         python2 \
     # gpg keys listed at https://github.com/nodejs/node#release-keys
-    && for key in \
-      4ED778F539E3634C779C87C6D7062848A1AB005C \
-      94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
-      74F12602B6F1C4E913FAA37AD3A89613643B6201 \
-      71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
-      8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
-      C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
-      C82FA3AE1CBEDC6BE46B9360C43CEC45C17AB93C \
-      DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-      A48C2BEE680E841632CD4E44F07496B3EB3C1762 \
-      108F52B48DB57BB0CC439B2997B01419BD92F80A \
-      B9E2F5981AA6E0CD28160D9FF13993A75599653C \
-    ; do \
-      gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
-      gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
-      gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
-    done \
+    && git clone https://github.com/nodejs/release-keys.git \
```

Comment:
Could probably skip the git dep, and maybe curl `https://github.com/nodejs/release-keys/archive/refs/heads/master.zip` to keep a smaller footprint than all the repo history (even though it's small now)

Reply:
hardcoding `master` seems inconvenient, what about just `git clone --depth 1`?

Reply:
I found it somehow odd to not pin the commit here or validate the downloaded artefact afterwards, maybe it would make sense to create a releases in https://github.com/nodejs/release-keys? While those are only keys, we are executing a random bash script here.